src/renderer/services/plugin-manager.js

```diff
@@ -247,43 +249,54 @@ class PluginManager {
               this._computedWatchers[computedName].getter = component.computed[computedName].bind(
                 componentContext(this)
               )
-              this._computedWatchers[computedName].run()
+
+              try {
+                this._computedWatchers[computedName].run()
+              } catch (error) {
+                console.error(error)
+              }
             }
 
             if (!component.computed || !component.computed[computedName]) {
               delete this.$options.computed[computedName]
-              this._computedWatchers[computedName].teardown()
+
+              try {
+                this._computedWatchers[computedName].teardown()
+              } catch (error) {
+                console.error(error)
+              }
+
               delete this._computedWatchers[computedName]
 
               for (const watcherId in this._watchers) {
                 if (this._watchers[watcherId].getter.name === computedName) {
-                  this._watchers[watcherId].teardown()
+                  try {
+                    this._watchers[watcherId].teardown()
+                  } catch (error) {
+                    console.error(error)
+                  }
+
                   break
                 }
               }
             }
           }
-          if (createdMethod) {
-            return createdMethod[0].apply(componentContext(this))
-          }
-        }]
 
-        // Fix context of all standard methods
-        vmComponent.options.methods = {}
-        for (const methodName of Object.keys(component.methods || {})) {
-          vmComponent.options.methods[methodName] = function () {
-            return component.methods[methodName].apply(componentContext(this), [...arguments])
+          if (component.created) {
+            return component.created.apply(componentContext(this))
```

Comment:
Isn't that unnecessary? since it is `vmComponent` that will be attached to the list of components.

Reply:
This line is needed otherwise the `created` hook that might exist in the component doesn't get called